pkg/limits/frontend/ring_test.go

```diff
@@ -50,7 +50,7 @@ func TestRingStreamUsageGatherer_GetStreamUsage(t *testing.T) {
 		name: "one stream",
 		getStreamUsageRequest: GetStreamUsageRequest{
 			Tenant:       "test",
-			StreamHashes: []uint64{1}, // Hash 1 maps to partition 1
+			StreamHashes: []uint64{0x1}, // Hash 0x1 maps to partition 0.
```

Comment:
I made these `0x` to make it easier for the read to differentiate the hashes for other counts in tests. I know `0x1` is the same as `1` 😄